src/Core/Logging/Extensions/ILogServiceExtensions.Internal.cs

```diff
@@ -0,0 +1,264 @@
+            string methodName = "",
+            [CallerMemberName] string extensionMethodName = "")
+        {
+            service.Internal(
+                timestamp,
+                state,
+                null,
+                null,
+                methodName,
+                extensionMethodName);
+        }
+
+        /// <summary>
+        /// Wrapper for <see cref="ILogServiceBase.Log{TState}(DateTimeOffset, LogLevel, TState, string)"./>
+        /// </summary>
+        public static void Internal(
+            this ILogService service,
+            DateTimeOffset timestamp,
+            Exception ex,
+            string methodName = "",
+            [CallerMemberName] string extensionMethodName = "")
+        {
+            service.Internal<object>(
+                timestamp,
+                null,
+                ex,
+                null,
+                methodName,
+                extensionMethodName);
+        }
+
+
+
+        /// <summary>
+        /// Wrapper for <see cref="ILogServiceBase.Log{TState}(LogLevel, TState, string)"./>
+        /// </summary>
+        public static void Internal<TState>(
+            this ILogService service,
+            TState message,
+            Exception ex,
+            string methodName = "",
+            [CallerMemberName] string extensionMethodName = "")
+        {
+            service.Internal(
+                DateTimeOffset.Now,
+                message,
+                ex,
+                null,
+                methodName,
+                extensionMethodName);
+        }
+
+        /// <summary>
+        /// Wrapper for <see cref="ILogServiceBase.Log{TState}(LogLevel, TState, string)"./>
+        /// </summary>
+        public static void Internal<TState>(
+            this ILogService service,
+            TState message,
+            string methodName = "",
+            [CallerMemberName] string extensionMethodName = "")
+        {
+            service.Internal(
+                DateTimeOffset.Now,
+                message,
+                null,
+                null,
+                methodName,
+                extensionMethodName);
+        }
+
+        /// <summary>
+        /// Wrapper for <see cref="ILogServiceBase.Log{TState}(LogLevel, TState, string)"./>
+        /// </summary>
+        public static void Internal(
+            this ILogService service,
+            Exception ex,
+            string methodName = "",
+            [CallerMemberName] string extensionMethodName = "")
+        {
+            service.Internal<object>(
+                DateTimeOffset.Now,
+                null,
+                ex,
+                null,
+                methodName,
+                extensionMethodName);
+        }
+
+
+
+        /// <summary>
+        /// Wrapper for <see cref="ILogServiceBase.Log{TState}(LogLevel, TState, Exception, Func{TState, Exception, string}, string)"./>
+        /// </summary>
+        private static void Internal<TState>(
```

Comment:
Codacy found an issue: [Method has 6 parameters, which is greater than the 4 authorized.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5762132)